app/views/templates/Layout.scala.html

```diff
@@ -15,108 +15,110 @@
  *@
 
 @import config.FrontendAppConfig
-@import uk.gov.hmrc.hmrcfrontend.views.config.StandardAlphaBanner
 @import views.html.helper.CSPNonce
 
 @this(
-    appConfig: FrontendAppConfig,
-    govukLayout: GovukLayout,
-    govukBackLink: GovukBackLink,
-    govukNotificationBanner: GovukNotificationBanner,
-    hmrcHead: HmrcHead,
-    hmrcStandardHeader: HmrcStandardHeader,
-    hmrcStandardFooter: HmrcStandardFooter,
-    hmrcTrackingConsentSnippet: HmrcTrackingConsentSnippet,
-    hmrcLanguageSelect: HmrcLanguageSelect,
-    hmrcTimeoutDialog: HmrcTimeoutDialog,
-    hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
-    hmrcScripts: HmrcScripts,
-    alphaBanner: StandardBetaBanner
+  appConfig: FrontendAppConfig,
+  govukLayout: GovukLayout,
+  govukBackLink: GovukBackLink,
+  govukNotificationBanner: GovukNotificationBanner,
+  hmrcHead: HmrcHead,
+  hmrcStandardHeader: HmrcStandardHeader,
+  hmrcStandardFooter: HmrcStandardFooter,
+  hmrcLanguageSelect: HmrcLanguageSelect,
+  hmrcTimeoutDialog: HmrcTimeoutDialog,
+  hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
+  hmrcScripts: HmrcScripts,
+  betaBanner: StandardBetaBanner
 )
 
-@(pageTitle: String, showBackLink: Boolean = true, timeout: Boolean = true, showSignOut: Boolean = true, refresh: Option[Int] = None, notificationBanner: Option[(String, String, String)] = None)(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
+@(pageTitle: String,
```

Comment:
is the formatting here manual or did you use scalafmt?

Reply:
Yes it has been manually formatted, scalafmt does not always formats files correctly, at least for me, have given a shot to see how it will format Layout but spitted out some random unrelated errors.